modules/ebi-metagenomics/cutadapt/main.nf

```diff
@@ -0,0 +1,56 @@
+
+process CUTADAPT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::cutadapt=4.6"
```

Comment:
Also, in nf-core they've stopped writing this and instead do this:
conda "${moduleDir}/environment.yml"
And the resepctive file:
name: cutadapt
channels:
  - conda-forge
  - bioconda
  - defaults
dependencies:
  - bioconda::cutadapt=3.4